### PR TITLE
BUG-119490

### PR DESCRIPTION
**Pull request by PankajWorks**
Changes :
1) for HDF if Ambari service configuration files are not present , copy them
2) Fail fast if dp version is not 1.3.X for CDH clusters

### CODE REVIEW

File: dp-cluster-setup-utility.py [hunk 1]
```diff
@@ -1497,7 +1500,21 @@ def update_profiler_agent_service_def(self, current_stack_version):
...
+  
+  def update_ambari_service_def(self, current_stack_version):
+    dest_services_base_dir = self._check_dir('/var/lib/knox/data-%s/services' % current_stack_version, 'service')
+    service_dir = '%s/ambari/0.2.2.0' % dest_services_base_dir
+    if os.path.isdir(service_dir):
+      print 'Service files already exist in %s' % service_dir
+    else:
+      os.makedirs(service_dir)
 
+    self._create_service_file(service_dir, 'rewrite.xml')
```
Comment (yhemanth) [inline dp-cluster-setup-utility.py]: _create_service_file is currently hardcoded to look for source files under profiler-agent directory. It should be parameterized. This PR should have failed on testing. Can you please confirm the changes with testing?

File: dp-cluster-setup-utility.py [hunk 2]
```diff
@@ -1497,7 +1500,21 @@ def update_profiler_agent_service_def(self, current_stack_version):
 
     self._chown_to_knox(service_dir)
     self._chown_to_knox(os.path.dirname(service_dir))
+  
+  def update_ambari_service_def(self, current_stack_version):
```
Comment (yhemanth) [inline dp-cluster-setup-utility.py]: There can be reuse between this and profiler agent copy I think. Please refactor the code appropriately. A lot of code seems to be the same, other than paths of the service files.

File: dp-cluster-setup-utility.py (hunk 1 above)
Comment (PankajWorks) [reply dp-cluster-setup-utility.py]: modified the code to pass service and version and then work accordingly

File: dp-cluster-setup-utility.py (hunk 2 above)
Comment (PankajWorks) [reply dp-cluster-setup-utility.py]: refactor the code by creating a new  function _execute_service_conf_file_copy_task

**Pull request merged**